src/utils/orgUtils.ts

```diff
@@ -48,8 +49,8 @@ export type SObjectCompactLayout = {
 export class OrgUtils {
     public static async getSobjects(): Promise<SObject[]> {
         try {
-            const org = await Org.create();
-            const conn = org.getConnection();
+            const conn =
```

Comment:
Changes in this file represent the entry points for the heart of the changes. Most of the other changes were either in the implementation of the backing Salesforce Extensions service connector code, or the supporting code in the tests.